app/facades/user_facade.rb

```diff
@@ -0,0 +1,40 @@
+class UserFacade
+  
+  def self.get_my_books(auth_token)
+    json = UserApiServices.get_users_books(auth_token, filter = nil)
+
+    json.map do |book_attributes|
+      BookPoro.new(book_attributes)
+    end
+  end
+
+  def self.add_book_response(book, auth_token, shelf = 2)
+    json = UserApiServices.add_a_book(book, auth_token, shelf)
+
+    if json == 200
+      'Book Added'
+    else
+      'Failed to add book'
+    end
+  end
+
+  def self.add_book_response(book, auth_token, shelf = 2)
```

Comment:
@LawrenceWhalen flagginge that I think this name needs to be `remove_book_response`?

Reply:
It did indeed, thank you!